bot/src/commands/revert-last-command.js

```diff
@@ -0,0 +1,54 @@
+import { get, post } from '../utils';
+import { Commands } from './index';
+
+function getCommandByName(name) {
```

Comment:
Ah, I see, that's a shame. I wonder why Babel doesn't make this additional property non-enumerable.